docs/.style/fsdocs-custom.sass

```diff
@@ -1 +1,154 @@
-//@import "./global-config.sass"
\ No newline at end of file
+@import "./global-config.sass"
+
+#fsdocs-nav
+  #fsdocs-logo
+    width: 90%
+    margin-top: 40px
+    border-style: none
+
+  input
+    margin-right: 20px
+    margin-top: 20px
+    margin-bottom: 20px
+    width: 93%
+    -webkit-border-radius: 0
+    border-radius: 0
+
+  font-size: 0.95rem
+
+  li.nav-header
+    padding-left: 0
+    color: $black
+    text-transform: none
+    font-size: 16px
+    margin-top: 9px
+    font-weight: bold
+
+  a
+    padding-left: 0
+    color: $gray
+
+blockquote 
+    margin: 0 1em 0 0.25em
+    margin-top: 0px
+    margin-right: 1em
+    margin-bottom: 0px
+    margin-left: 0.25em
+    padding: 0 .75em 0 1em
+    border-left: 1px solid $gray
+    border-right: 0px solid $gray
+
+    
+#fsdocs-content
+
+    h1 a
```

Comment:
Maybe write this in one selector instead of repeating yourself.
Something like:
```sass
h1, h2, h3, h4, h5, h6
  a
    color: $black
```